app/src/pages/authoring/index.tsx

```diff
@@ -66,20 +86,39 @@ export default function AuthoringPage() {
   };
 
   const draftMutation = trpc.draft.createDraft.useMutation({
-    onSuccess: data => {
-      successNotification(data, false);
+    onSuccess: (data, variables) => {
+      successNotification(variables.resourceType, false, false);
+      router.push(`authoring/${resourceType}/${data.draftId}`);
     },
-    onError: e => {
-      errorNotification(e.message, false);
+    onError: (e, variables) => {
+      errorNotification(variables.resourceType, e.message, false, false);
     }
   });
 
-  const draftFromArtifactMutation = trpc.service.convertArtifactById.useMutation({
-    onSuccess: data => {
-      successNotification(data, true);
+  const draftChildMutation = trpc.service.draftChild.useMutation({
```

Comment:
Might be worth adding a comment to this function regarding the assumptions we make with failing child artifacts. This will help us in the future if we need to look back and see where the assumptions come into play with the actual code.